test/test_workflow_Run2016B.py

```diff
@@ -63,7 +66,6 @@
 #         os.remove(fname)
 
 # check drifts and create IOV
-#filename = 'BS_result_PiltBeam2021_byIOV/beamspot_Express_PilotBeam2021_.txt'
```

Comment:
sorry i didn't notice this before but this is used as well when creating 1 txt for each IOV, so it should be resotred as well.
Just FYI it is used in line 80:
`fname = filename.replace('_.txt', '_'+str(count)+'.txt')`

Reply:
Done ;)